preview/src/index.html

```diff
@@ -2,9 +2,20 @@
 <html>
 
 <head>
-    <meta charset="utf-8">
     <title>Flow2</title>
+    <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+        html, body {
```

Comment:
Would it make sense to move this to the root component, perhaps by wrapping the `#editor-container` `div` in another `div`?

Reply:
(that outer `div` being the `#flow-editor` `div`)